packages/SystemsThinkingWorkbench-Core.package/STWConnection.class/instance/drawArrowOn..st

```diff
@@ -0,0 +1,14 @@
+drawing
+drawArrowOn: aCanvas 
```

Comment:
Maybe add Documentation about what the three lines represent and how they are calculated